CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Add support for stopping connectors according to [Strimzi Proposal #54](https://github.com/strimzi/proposals/blob/main/054-stopping-kafka-connect-connectors.md)
 * Allow manual rolling of Kafka Connect and Kafka Mirror Maker 2 pods using the `strimzi.io/manual-rolling-update` annotation (supported only when `StableConnectIdentities` feature gate is enabled) 
 * Make sure brokers are empty before scaling them down
+* Update Cruise Control to 2.5.128
```

Comment:
We have never highlight this kind of update in the CHANGELOG. Tbh I don't think it's so important to do so. @scholzj wdyt?

Reply:
yeah I wasn't sure because of this
https://github.com/strimzi/strimzi-kafka-operator/pull/8525
https://github.com/strimzi/strimzi-kafka-operator/pull/9025

Reply:
There are many dependency updates due to CVEs on every release, so this would create a lot of changelog noise IMO. What we could do is to add a "cve-fix" label so that one can easily spot them on the release issue list. Wdyt?

Reply:
yeah I wasn't sure as it was added until 0.29.0 like here
https://github.com/strimzi/strimzi-kafka-operator/blob/main/CHANGELOG.md#0290

Reply:
I do not add it when I update CC. But we have it in some past change logs. So I'm personally fine with keeping this 🤷 .

> There are many dependency updates due to CVEs on every release, so this would create a lot of changelog noise IMO. What we could do is to add a "cve-fix" label so that one can easily spot them on the release issue list. Wdyt?

This creates a false impression that the labeled PRs give you an exact idea about the fixed CVEs. That would not be the case because in many cases CVEs are fixed without knowing. So I'm not sure that is a good idea.

Reply:
> I do not add it when I update CC. But we have it in some past change logs. So I'm personally fine with keeping this 🤷 .

We should try to have consistency which also helps users to understand what's going on imho. So if we take this line this time, let's remember to do the same for any future CC updates otherwise the users reading the CHANGELOG can think that this was the latest update and no more happened (while we are just omitting it in the CHANGELOG).

Reply:
> This creates a false impression that the labeled PRs give you an exact idea about the fixed CVEs. That would not be the case because in many cases CVEs are fixed without knowing. So I'm not sure that is a good idea.

That's true. In the end, it is a major component, so I'm good with leaving it.